library/Icinga/Web/Navigation/ConfigMenu.php

```diff
@@ -222,6 +228,20 @@ protected function createHealthBadge()
         return $stateBadge;
     }
 
+    protected function createMigrationBadge(): ?StateBadge
+    {
+        $mm = MigrationManager::instance();
+        $count = $mm->count();
```

Comment:
This may throw. The menu should still be usable in case some extension/hook throws. So please add a try-catch here and just log the error. There should be no badge returned in such a case of course:

```suggestion
        try {
            $mm = MigrationManager::instance();
            $count = $mm->count();
        } catch (Throwable $e) {
            Logger::error('Failed to load pending migrations: %s', $e);
            $count = 0;
        }
```